src/api/registration/EditorRegistration.ts

```diff
@@ -41,8 +45,16 @@ export interface EditorRegistration<PARAMS = EmptyParameters> extends Partial<Ed
     /**
      * Should this be a dry run editor: That is,
      * should it wait for the build result to determine whether to raise a pull request
-     * or issue. Default is no.
+     * or issue. Default is editorCommand, which creates a branch and a PR.
      */
-    dryRun?: boolean;
+    editorCommandFactory?: EditorCommandFactory<PARAMS>;
 
 }
+
+export type EditorCommandFactory<PARAMS> = (
```

Comment:
Would be great to have a comment on this type

Reply:
ok but I don't have a deep understanding of the arguments; I don't think it's really a clean abstraction, but it is a step in the direction we needed.